src/Packetery/Module/Framework/HookTrait.php

```diff
@@ -15,6 +15,14 @@
  * @package Packetery
  */
 trait HookTrait {
+	public function addAction( string $hookName, callable $callback, int $priority = 10, int $acceptedArgs = 1 ): ?bool {
+		return add_action( $hookName, $callback, $priority, $acceptedArgs );
+	}
+
+	public function addFilter( string $hookName, callable $callback, int $priority = 10, int $acceptedArgs = 1 ): void {
```

Comment:
normálně bych byl byl proti těm defaultním hodnotám v parametrech, akorát to vede k tomu, že je snadno přehledneš, protože duplicitně je tam zase posílat je za mě ve většině případech blbost, ale tím, že už to tak bohužel mají ve WC, tak to tak asi nechejme. Aspo si ošetříme, kdyby v nějaké verzi ty hodnoty náhodou změnili